crates/compute/src/memory.rs

```diff
@@ -184,6 +185,14 @@ impl<'a, F, Mem: ComputeMemory<F>> SubfieldSlice<'a, F, Mem> {
 	pub fn new(slice: Mem::FSlice<'a>, tower_level: usize) -> Self {
 		Self { slice, tower_level }
 	}
+
+	/// Returns the length of the slice in terms of the number of subfield elements it contains.
+	pub fn subfield_len(&self) -> usize
```

Comment:
IMO this should just be `len()` since the struct is already `SubfieldSlice`. You can leave as is and resolve if you find this less confusing though.